INSTALL_DOCKER.rst

```diff
@@ -93,7 +93,7 @@ To enable your user to access both Docker and KVM you will need to add them to t
 
     $ sudo usermod -G kvm,docker -a <USERNAME>
 
-Then run the following commands to add those group ids to your active login session:
+The easiest way to get these new group memberships to take effect is to log out and log back in again as \<USERNAME\>. Alternatively, you can run the following commands to add those group ids to your active login session, but be aware each of these commands drops you into a new sub-shell, so you need two extra `exits` at the end of your session to leave your shell:
 
   ::
 
```

Comment:
@gorozco1 I'm more than happy to. It will make it simpler, and I had considered already. Anybody else disagree - if not I'll get on it later...